firefox-ios/Shared/DateGroupedTableData.swift

```diff
@@ -12,84 +12,72 @@ private func getDate(dayOffset: Int) -> Date {
 }
 
 public struct DateGroupedTableData<T: Equatable> {
-    let todayTimestamp = getDate(dayOffset: 0).timeIntervalSince1970
-    let yesterdayTimestamp = getDate(dayOffset: -1).timeIntervalSince1970
-    let lastWeekTimestamp = getDate(dayOffset: -7).timeIntervalSince1970
-    let lastMonthTimestamp = getDate(dayOffset: -30).timeIntervalSince1970
+    // Timestamps at which we want the data to be split
+    var timestamps: [TimeInterval] = []
```

Comment:
The changes in this file feel like they all easily tested and we should have tests for them to know things are operating exactly as we're expecting. :)